From 5240972d8df21173492ba37dc3ea3ea0b1a99623 Mon Sep 17 00:00:00 2001
From: Lennart Poettering <lennart@poettering.net>
Date: Mon, 29 Apr 2019 16:10:51 +0200
Subject: [PATCH 22/29] units: drop reference to sushell man page

sushell was a Fedoraism, and has been removed since. Hence our upstream
unit files shouldn't reference it either.

(cherry picked from commit 6dc14d73664390682d47d7e5bcbdbb362d04f623)
---
 units/debug-shell.service.in | 1 -
 1 file changed, 1 deletion(-)

diff --git a/units/debug-shell.service.in b/units/debug-shell.service.in
index 1127e68b63..9f3868e106 100644
--- a/units/debug-shell.service.in
+++ b/units/debug-shell.service.in
@@ -9,7 +9,6 @@
 
 [Unit]
 Description=Early root shell on @DEBUGTTY@ FOR DEBUGGING ONLY
-Documentation=man:sushell(8)
 Documentation=man:systemd-debug-generator(8)
 DefaultDependencies=no
 IgnoreOnIsolate=yes
-- 
2.21.0

